libs/blocks/tabs/tabs.js

```diff
@@ -128,6 +136,13 @@ function initPaddles(tabList, tabPaddles) {
     }
   });
 
+  tabList.addEventListener('scroll', () => {
+    tabPaddles.setAttribute(
+      'aria-valuenow',
+      ((tabList.scrollLeft / (tabList.scrollWidth - tabList.clientWidth)) * 100).toFixed(0),
+    );
+  });
```

Comment:
Nice! But let's also debounce this. There should be an utility in `utils.js` to help out for the moment.

Reply:
Found it and made the adjustment 👍